ajax/libs/Embetty/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Embetty",
+  "description": "Embetty displays remote content like tweets or videos without compromising your privacy.",
+  "version": "1.2.0",
+  "license": "MIT",
+  "author": {
+    "name": "Philipp Busse",
+    "email": "pmb@heise.de"
+  },
+  "keywords": [
+    "tweets",
+    "video",
+    "embed"
+  ],
+  "homepage": "https://heiseonline.github.io/embetty/",
+  "filename": "embetty.min.js",
+  "npmName": "@heise/embetty",
+  "npmFileMap": [
+    {
+      "basePath": "@heise/embetty/dist",
+      "files": [
+        "*/.js"
```

Comment:
```suggestion
        "*.js"
```
The only two files in `dist` are `embetty.js` and `report.html`. Assuming we don't need the html file, this glob can just be `*.js`.